docs/docs/developers/contracts/references/storage/private_state.md

```diff
@@ -42,19 +42,19 @@ A note should implement the following traits:
 
 The interplay between a private state variable and its notes can be confusing. Here's a summary to aid intuition:
 
-A private state variable (of type `Singleton`, `ImmutableSingleton` or `Set`) may be declared in storage.
+A private state variable (of type `PrivateMutable`, `PrivateImmutable` or `Set`) may be declared in storage.
```

Comment:
Should the `Set` also be renamed? If looking above, we have `PrivateMutable<NoteType`, `PrivateImmutable<NoteType>` and then `Set<NoteType>` which can make it seem like we also have a public set as it is not explicitly private.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4739/commits/caa14a38906c1f0e0ea76bacf8a1b29bd7ea5a86